git-send-email.perl

```diff
@@ -299,7 +304,7 @@ sub signal_handler {
                     "dump-aliases" => \$dump_aliases);
 usage() unless $rc;
 die "--dump-aliases incompatible with other options\n"
-    if !$help and $dump_aliases and @ARGV;
+    if (!$help) and $dump_aliases and @ARGV;
```

Comment:
If you're saying that conforming to PBP isn't worthwhile, and we should drop the patch, then that's fine.  PBP isn't everyone's cup of tea.